### ISSUE TITLE
😱😱😱😱 Can't do any normal game 😱😱😱😱

### ISSUE THREAD

**Issue by Getta91** [mmtrt/leagueoflegends#57]
Hi ! I was playing normally for a while but recently after a fresh reinstall my game does'nt work. Let me explain: I can do practice game, create or join custom game. But with ranked game and normal game (draft and blindpick), I can't go into the game. My game is loading to 100% after like 15 min and then it stuck itself. It is not my connection, I have already checked it.  Please help me I have got a leaverbuster T_T.
Thanks for your futur help.

![image](https://user-images.githubusercontent.com/50636257/59848351-357ee780-9365-11e9-8968-8fd75ee01a4e.png)
<3 From a Vel'koz lover <3

**Comment by mmtrt** [comment]
Maybe try to post terminal log or post info about the snap info wine-platform-runtime and snap info  wine-platform-4-staging

**Comment by Getta91** [comment]
blotemmanuel@blotemmanuel-HP-ProBook-6460b:~$ snap info wine-platform-runtime
name:    wine-platform-runtime
summary: wine platform runtime snap that provides WINE runtime libraries for
  wine-platform-x-[stable,devel,staging] snaps
publisher: Taqi Raza (mmtrt)
license:   unset
description: |
  This snap provides WINE runtime base via content interface to be used by
  other snaps that are depended on WINE so snap developers that create WINE
  snaps can leverage this base snap to connect their WINE snaps.
  
  Features:
  
  Snap provides WINE runtime libraries that can be used by these snaps.
  
  • wine-platform-3-stable
  • wine-platform-4-stable
  • wine--platform-4-devel
  • wine--platform-4-staging
  
  Currently powering the following snaps that are using this snap:
  
  • anifx
  • anituner
  • cncra
  • cncra2yr
  • cnctsun
  • fooba2000
  • irfanview
  • leagueoflegends
  • liveforspeed
  • multicommander
  • notepad-plus-plus
  • notepad3
  • pspad
  • resourcehacker
  • utorrent
  • winsnap
  
  Note:  This snap does not provides any ability to run WIN applications from
  itself but it only shares libraries and other pkg releases PATHS that are
  available via content interface.
  
  Support me: https://bit.ly/2XkT2bl
snap-id:      l4K2MbtPjiGgToWYH70ePXKaGQoF1VXF
tracking:     candidate
refresh-date: 2 days ago, at 10:23 CEST
channels:
  stable:    v1.0 2019-06-06  (6) 226MB -
  candidate: v1.0 2019-06-07  (7) 226MB -
  beta:      v1.0 2019-06-17 (17) 227MB -
  edge:      v1.0 2019-06-16 (15) 227MB -
installed:   v1.0             (7) 226MB -

**Comment by Getta91** [comment]
blotemmanuel@blotemmanuel-HP-ProBook-6460b:~$ snap info wine-platform-4-staging
name:      wine-platform-4-staging
summary:   wine platform snap it provides staging WINE 4 for snaps
publisher: Taqi Raza (mmtrt)
license:   unset
description: |
  This snap creates a WINE staging 4 via content sharing to be used by other snaps that are using
  WINE.
  
  Support me: https://bit.ly/2XkT2bl
snap-id:      t0SWlyjLrOQqAKWoCA0YBGCV0wLev1lD
tracking:     candidate
refresh-date: yesterday at 12:25 CEST
channels:
  stable:    4.9  2019-06-07  (5) 145MB -
  candidate: 4.9  2019-06-07  (7)  95MB -
  beta:      4.10 2019-06-18 (12)  94MB -
  edge:      4.9  2019-06-16 (10) 145MB -
installed:   4.9              (7)  95MB -

**Comment by Getta91** [comment]
I can't post the log because of the leaverbuster

**Comment by mmtrt** [comment]
maybe try from beta branch also don't try to play ranked games on likely game breaks. 
```
snap refresh --beta wine-platform-runtime
snap refresh --beta wine-platform-4-staging
````

**Comment by Getta91** [comment]
i tried but now i have a crash dump after champ select

**Comment by mmtrt** [comment]
try it winxp

**Comment by Getta91** [comment]
It was the first thing I did. When i saw the bug, i tried with win 7 but it didn't do anything

**Comment by mmtrt** [comment]
maybe try to kill wineserver before launching game helps it bug happens

leagueoflegends.wineserver

**Comment by Getta91** [comment]
does this command kill wineserver? :

leagueoflegends.wineserver

**Comment by mmtrt** [comment]
Yes it does

**Comment by Getta91** [comment]
no league just crashes

**Comment by mmtrt** [comment]
use wine appimage then if snap does not work for you.
https://github.com/mmtrt/Wine_Appimage/releases/tag/release-lol-patched

**Comment by Getta91** [comment]
Will I still need to do the different update of the league snap?

**Comment by Getta91** [comment]
Thanks now I can join the game... But I have a very ssstttrrraannnggeee issue. I don't see any text in game. Only the shop and the chat are working fine. I can't see levels of players, my gold or usernames.

**Comment by mmtrt** [comment]
Install directx in appimage wineprefix

**Comment by Getta91** [comment]
But how can I do this please ?

**Comment by mmtrt** [comment]
If you have winetricks installed then run command
 WINEPREFIX=$HOME/.wine-appimage-lol winetricks directx9

**Comment by Getta91** [comment]
Thanks a lot

**Comment by Getta91** [comment]
Hi @mmrt! I know this issue is closed but i found a fix and I am sure it can help others. So the first I did, was uninstalling wine with this command : "sudo apt purge wine* "(found this on reddit). Then i reinstalled wine by following the steps of winehq website. BUT, I installed the staging version this time. The fact is that lol snap couldn't run with the stable version. Now it's working nice! Thanks and hope this would help!
